test/unit/config.js

```diff
@@ -11,111 +11,52 @@ describe('Config', function () {
   before('instanciate config', configHelpers.createConfig)
   after('clean config directory', configHelpers.cleanConfig)
 
-  describe('saveConfig', () =>
+  describe('persist', function () {
     it('saves last changes made on the config', function () {
-      this.config.devices['new-cozy2'] = {
-        deviceName: 'new-cozy2',
-        passphrase: 'passphrase',
-        url: 'none'
-      }
-      this.config.save()
+      this.config.setCozyUrl('http://cozy.local:8080/')
+      this.config.persist()
       let conf = new Config(path.join(this.syncPath, '.cozy-desktop'))
-      should.exist(conf.devices['new-cozy2'])
-    })
-  )
-
-  describe('getDefaultDeviceName', function () {
-    it('returns devicename from args', function () {
-      process.argv = ['arg1', '-d', 'test']
-      let name = this.config.getDefaultDeviceName()
-      name.should.equal('test')
-    })
-    it('returns default devicename when no args', function () {
-      process.argv = []
-      let name = this.config.getDefaultDeviceName()
-      name.should.equal('tester')
+      conf.getCozyUrl().should.equal('http://cozy.local:8080/')
```

Comment:
I started using the following pattern in new code:

```javascript
should(conf.getCozyUrl()).equal(...)
```

Pros:
- no monkey-patching involved...
- ... which makes it more Flow-friendly

Please let me now in case you disagree, so I stop using it everywhere.